lake-manifest.json

```diff
@@ -19,7 +19,12 @@
     "rev": "071464ac36e339afb7a87640aa1f8121f707a59a",
     "name": "aesop",
     "inputRev?": "master"}},
-  {"path": {"name": "GameServer", "dir": "./../lean4game/server"}},
+  {"git":
+   {"url": "https://github.com/leanprover-community/lean4game.git",
+    "subDir?": "server",
+    "rev": "98597c1522de78c64fc5f9bc4a575ba7f43c7c4f",
+    "name": "GameServer",
+    "inputRev?": "main"}},
```

Comment:
@abentkamp ist das ein indikator, dass wir das lake-manifest nicht in git haben sollten? Oder sollte die `git`-version default drin sein und wir prüfen, dass wir das nicht versehentlich mit der lokalen dependency überschreiben?

Reply:
Hm, ja vielleicht sollte das manifest in die gitignore.